### DISCUSSION TITLE
1.2.0 release

### DISCUSSION THREAD

**Discussion by evpobr** [libsndfile/libsndfile#896] [General]
So, it is time i guess.

@arthurt , @SoapGentoo , are there any important things that need to be done before the release?

**Comment by SoapGentoo** [comment]
not that I'm aware of

**Comment by arthurt** [comment]
Okay, had some time off (I was sick actually), and got the three little outstanding things in. No objections to another release.

**Comment by evpobr** [comment]
Excellent. Unfortunately, I'm still busy at work. @SoapGentoo , maybe next weekend?

**Comment by SoapGentoo** [comment]
sure, let me know once you're ready

**Comment by evpobr** [comment]
@SoapGentoo , https://github.com/libsndfile/libsndfile/tree/release-1.2.0.

Tomorrow is OK?

**Reply by SoapGentoo** [reply]
yes, let's do it tomorrow

**Reply by evpobr** [reply]
Start when you ready..

**Reply by SoapGentoo** [reply]
tarball and signature are up

**Reply by evpobr** [reply]
Good. Building Windows binaries.

**Reply by evpobr** [reply]
[libsndfile-1.2.0-win32.zip](https://github.com/libsndfile/libsndfile/files/10299907/libsndfile-1.2.0-win32.zip)
[libsndfile-1.2.0-win64.zip](https://github.com/libsndfile/libsndfile/files/10299908/libsndfile-1.2.0-win64.zip)

**Reply by SoapGentoo** [reply]
uploaded with sigs, can you just push your `_config.yaml` commit to master?

**Reply by evpobr** [reply]
Done.

**Comment by evpobr** [comment]
Thanks everybody!
